hawc/apps/lit/urls.py

```diff
@@ -50,11 +50,6 @@
         views.RefsByTagJSON.as_view(),
         name="refs_json",
     ),
-    url(
-        r"^assessment/(?P<pk>\d+)/references/download/$",
-        views.RefDownloadExcel.as_view(),
-        name="ref_download_excel",
```

Comment:
looks like this is also in `reference_list.html` and `reference_visual.html` (ctrl-shift-f for ref_download_excel in vscode)

Reply:
Updated reference_list.html and reference_visual.html to use new endpoint.